client/src/document/toolbar/flaws.tsx

```diff
@@ -219,6 +219,7 @@ function Flaws({
 
   const isReadOnly = !CRUD_MODE_HOSTNAMES.includes(window.location.hostname);
 
+  const filePath = doc.source.folder + "/" + doc.source.filename;
```

Comment:
```suggestion
  // Note! This will work on Windows. The filename can be sent to 
  // the server in POSIX style and the `open-editor` program will make
  // this work for Windows automatically. 
  const filePath = doc.source.folder + "/" + doc.source.filename;
```